package.json

```diff
@@ -4,7 +4,8 @@
   "description": "ESLint config following Wikimedia code conventions.",
   "main": ".eslintrc.json",
   "scripts": {
-    "test": "eslint test/fixtures/*.js && node test/index.js"
+    "test": "eslint . && node test/valid.js && node test/invalid.js",
```

Comment:
You can't `eslint .` because invalid.js is caught by this (and, reasonably, doesn't pass).